common/src/main/java/de/maxhenkel/voicechat/voice/client/KeyEvents.java

```diff
@@ -55,9 +57,11 @@ public static void registerKeyBinds() {
         KEY_GROUP = ClientCompatibilityManager.INSTANCE.registerKeyBinding(new KeyMapping("key.voice_chat_group", GLFW.GLFW_KEY_G, "key.categories.voicechat"));
         KEY_TOGGLE_RECORDING = ClientCompatibilityManager.INSTANCE.registerKeyBinding(new KeyMapping("key.voice_chat_toggle_recording", InputConstants.UNKNOWN.getValue(), "key.categories.voicechat"));
         KEY_ADJUST_VOLUMES = ClientCompatibilityManager.INSTANCE.registerKeyBinding(new KeyMapping("key.voice_chat_adjust_volumes", InputConstants.UNKNOWN.getValue(), "key.categories.voicechat"));
+        KEY_DISPLAY_NAMETAGS = ClientCompatibilityManager.INSTANCE.registerKeyBinding(new KeyMapping("key.toggle_nametags", InputConstants.UNKNOWN.getValue(), "key.categories.voicechat"));
+        KEY_ON_TALK_NAMETAGS = ClientCompatibilityManager.INSTANCE.registerKeyBinding(new KeyMapping("key.toggle_on_talk_nametags", InputConstants.UNKNOWN.getValue(), "key.categories.voicechat"));
 
         ALL_KEYS = new KeyMapping[]{
-                KEY_PTT, KEY_WHISPER, KEY_MUTE, KEY_DISABLE, KEY_HIDE_ICONS, KEY_VOICE_CHAT, KEY_VOICE_CHAT_SETTINGS, KEY_GROUP, KEY_TOGGLE_RECORDING, KEY_ADJUST_VOLUMES
+                KEY_PTT, KEY_WHISPER, KEY_MUTE, KEY_DISABLE, KEY_HIDE_ICONS, KEY_VOICE_CHAT, KEY_VOICE_CHAT_SETTINGS, KEY_GROUP, KEY_TOGGLE_RECORDING, KEY_ADJUST_VOLUMES, KEY_DISPLAY_NAMETAGS, KEY_ON_TALK_NAMETAGS
```

Comment:
There is no reason to have this as a keybind. People won't change this setting that often.